src/mixins/changeStateMixin.js

```diff
@@ -222,7 +246,8 @@ export const changeStateMixin = {
       ) {
         this.setAlert({
           alertShow: true,
-          alertMessage: 'Flow run set for cancel.  This may take 30 seconds.',
+          alertMessage:
+            'Your flow run has been cancelled; please allow up to 30 seconds for this to take effect.',
```

Comment:
one last message update similar to the above - apologies for the scattered review